markdown/releasenotes/index.md

```diff
@@ -1,85 +1,146 @@
 # Iris Release Notes
 
+## Iris 3.0 Release Notes
+
+Iris 3.0b1 introduced a new infrastructure for visualizing and fitting
```

Comment:
Let's just condensate everything in Iris 3.0. No need to break down changes by betas.